### PR TITLE
Issue 1015

### PR DESCRIPTION
**Pull request by quanvo87**
Add HSCAN, BITFIELD, KEYS, RANDOMKEY, SCAN, SORT, TOUCH, and TYPE.

## Checklist:

- [ ] I have submitted a [CLA form](https://github.com/IBM-Swift/CLA)
- [ ] If applicable, I have updated the documentation accordingly.
- [x] If applicable, I have added tests to cover my changes.

### CODE REVIEW

**Comment by quanvo87** [comment]
@shmuelk This includes the GEO commands I added

File: Sources/RedisMulti+Basic.swift [hunk 1]
```diff
@@ -54,6 +54,77 @@ extension RedisMulti {
...
+            case .overflow(let overflow):
+                command.append("OVERFLOW")
+                switch(overflow) {
+                case .WRAP:
+                    command.append(BitfieldSubcommand.BitfieldOverflow.WRAP.rawValue)
+                case .SAT:
+                    command.append(BitfieldSubcommand.BitfieldOverflow.SAT.rawValue)
+                case .FAIL:
+                    command.append(BitfieldSubcommand.BitfieldOverflow.FAIL.rawValue)
+                }
```
Comment (shmuelk) [inline Sources/RedisMulti+Basic.swift]: Why do you need the switch here at all? Why not just write:

     command.append(overflow.rawValue)

Isn't overflow of the type `BitfieldSubcommand.BitfieldOverflow`?

File: Sources/Redis+Basic.swift [hunk 2]
```diff
@@ -58,6 +58,88 @@ extension Redis {
...
+            case .overflow(let overflow):
+                command.append("OVERFLOW")
+                switch(overflow) {
+                case .WRAP:
+                    command.append(BitfieldSubcommand.BitfieldOverflow.WRAP.rawValue)
+                case .SAT:
+                    command.append(BitfieldSubcommand.BitfieldOverflow.SAT.rawValue)
+                case .FAIL:
+                    command.append(BitfieldSubcommand.BitfieldOverflow.FAIL.rawValue)
+                }
```
Comment (shmuelk) [inline Sources/Redis+Basic.swift]: Why do you need the switch here at all? Why not just write:

     command.append(overflow.rawValue)

Isn't overflow of the type `BitfieldSubcommand.BitfieldOverflow`?

File: Sources/Redis+Geo.swift [hunk 3]
```diff
@@ -0,0 +1,324 @@
...
+    /// - parameter error: Non-nil if error occurred.
+    public func geoposArrayOfMembers(key: String, members: [String], callback: (_ result: [RedisResponse?]?, _ error: NSError?) -> Void) {
+        var command = ["GEOPOS", key]
+        for member in members {
+            command.append(member)
+        }
+        issueCommandInArray(command) { (response) in
+            redisArrayResponseHandler(response: response, callback: callback)
+        }
+    }
```
Comment (shmuelk) [inline Sources/Redis+Geo.swift:154]: Instead of returning `[RedisResponse?]?` Why not return `[(Double, Double)?]?` ? That is an array of optional tuples.

**Comment by shmuelk** [comment]
LGTM

**Pull request merged**